### PR TITLE
push a version tag for jenkins centos images

### PR DESCRIPTION
**Pull request by bparees**

### CODE REVIEW

**Comment by stevekuznetsov** [comment]
is this the point where we make a for loop?

**Comment by bparees** [comment]
obviously not. :)

File: sjb/config/test_cases/push_jenkins_images.yml [hunk 1]
```diff
@@ -32,22 +32,32 @@ extensions:
         sudo chmod a+rw /tmp/.docker/config.json
         export DOCKER_CONFIG=/tmp/.docker
 
+        version_tag=`cat release.version`
```
Comment (stevekuznetsov) [inline sjb/config/test_cases/push_jenkins_images.yml]: use `$()`

Comment (bparees) [reply sjb/config/test_cases/push_jenkins_images.yml]: thanks, that worked.

**Pull request merged**